testdata/jsight_0.3/ser-to-openapi/req.japi.schema_notation/req.japi.schema_notation.jsight_0.3/jsight_notation_in_directive_body.openapi.todo

```diff
@@ -0,0 +1,127 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "",
+    "version": ""
+  },
+  "paths": {
+    "/notation/in/request/body/": {
+      "post": {
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "type": "object",
+                "properties": {
+                  "name": {
+                    "type": "string"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "responses": {
+          "200": {
```

Comment:
There is a mistake here. Any response must be allowed:

```
          "responses": {
            "default": {
              "description": "",
              "content": {
                "*/*": {}
              }
            }
          }
```

Reply:
Fixed as suggested here.